### PR TITLE
Tighten "cnf" claim description

### LINKED ISSUE #52: Describe how `cnf` is allowed to be used for "binding"
How should a verifier interpret the `cnf` claim when it is present in a `vc-jwt-2.0`.

### PR DESCRIPTION
**Pull request by selfissued**
Fixes #52

***
<a href="https://pr-preview.s3.amazonaws.com/selfissued/vc-jose-cose/pull/155.html" title="Last updated on Sep 20, 2023, 9:39 PM UTC (3ff821a)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-jose-cose/155/79f5486...selfissued:3ff821a.html" title="Last updated on Sep 20, 2023, 9:39 PM UTC (3ff821a)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -465,14 +465,9 @@ <h2>cnf</h2>
       If <code>cnf</code> is present in the <a data-cite="RFC7515#section-4.1">Protected Header</a>
       or the <a data-cite="RFC7519#section-4.1.1">JWT Claims </a>,
       a <a data-cite="VC-DATA-MODEL#dfn-verifier">verifier</a> can use this parameter
```
Comment (andresuribe87) [inline index.html:467]:
```suggestion
      a <a data-cite="VC-DATA-MODEL#dfn-verifier">verifier</a> MAY use this parameter
```

**Review by andresuribe87 — APPROVED** [review]
Minor nit.

File: index.html [hunk 2]
```diff
@@ -465,14 +465,9 @@ <h2>cnf</h2>
...
-      to obtain a <a data-cite="RFC7517#section-4">JSON Web Key</a> to use in the 
+      to identify a proof-of-possesion key in the manner described in [[rfc7800]] to use in the
```
Comment (TallTed) [inline index.html:468]:
```suggestion
      to identify a proof-of-possesion key in the manner described in [[rfc7800]] for use in the
```

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/2017/vc/WG/Meetings/Minutes/2023-09-14-vcwg#section6-5) on 2023-09-14

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

#### 6.5. Tighten "cnf" claim description (pr vc-jose-cose#155)

_See github pull request [vc-jose-cose#155](https://github.com/w3c/vc-jose-cose/pull/155)._




**Michael Jones:** the deletion is removing sentence that never made sense.  

**Brent Zundel:** just a day old, please review. adds a normative reference to 7800. not normative.  

</details>

File: index.html (hunk 1 above)
Comment (OR13) [reply index.html:467]: @selfissued ^ can you comment on this suggestion?

**Review by OR13 — APPROVED** [review]

**Pull request merged**